### PR TITLE
Add script to compute additional metrics

### PR DESCRIPTION
**Pull request by phucdev**
The training scripts produce a `generated_predictions.jsonl` file that contains the predicted summaries along with the documents and reference summaries.
I added a `compute_metrics.py` script that takes this file and computes ROUGE, METEOR, BERTScore and BLANC-help.

ROUGE, METEOR and BERTScore already exist in https://huggingface.co/evaluate-metric.
For BLANC I added a Hugging Face Space so we can also load it via `evaluate`: https://huggingface.co/spaces/phucdev/blanc_score

BLANC is a reference-free metric. It does not require a reference summary. Instead it measures how much a summary helps a pre-trained language model (like BERT) perform a language understanding task, such as filling in masked words in a document.
There are two variants of BLANC. We use BLANC-help. The summary is concatenated with document sentences during the inference task. The BLANC-help score is defined as the difference in accuracy between unmasking tokens with the summary and with filler text (same length as the summary but consisting of period symbols). It measures how much the summary boosts the model’s performance on masked tokens.

I also updated the README.md with more instructions.

### CODE REVIEW

**Comment by phucdev** [comment]
@clement-gillet Do we need to average: 
- the precision, recall and f1 scores for BERTScore,
- the scores from BLANC-help?

For now the script produces individual scores for each example.

**Review by clement-gillet — APPROVED** [review]
Nice job ! Yes, I'm doing it this morning.

**Comment by phucdev** [comment]
1. We could directly average the scores in the `compute_metrics.py` script.

2. Regarding your original comment: We could directly integrate METEOR and BERTScore in the training script. However it is non-trivial to also integrate BLANC into the `compute_metrics` method because BLANC does not use the reference summaries, but the original documents. In my opinion this would require too many changes affecting the `Trainer` as we we would have to pass the original document along with the predicted & reference summary to `compute_metrics`.

**Comment by clement-gillet** [comment]
1. Yes, I've done that
2. I agree
